spec/services/forecast_history_spec.rb

```diff
@@ -232,13 +238,19 @@ def history_entries
         ])
       end
 
-      it "adds a revision with a zero value" do
-        history.set_value(0)
+      context "with a zero value" do
+        it "adds a revision" do
+          history.set_value(0)
 
-        expect(history_entries).to eq([
-          ["original", 1, 2015, 10],
-          ["revised", 2, 2015, 0]
-        ])
+          expect(history_entries).to eq([
+            ["original", 1, 2015, 10],
+            ["revised", 2, 2015, 0]
+          ])
+        end
+
+        it "returns a _Forecast_" do
```

Comment:
again, no biggie but any reason for the `_` around `Forecast` here? Is that a convention we use for class name?

Reply:
If we were producing documentation with rdoc, the underscore would be formatted as `<em>`tags.

That being said, I don't think we do this anywhere else in our specs, nor have I ever used the specs to generate docs with rdoc.

Is it worth removing it everywhere in the file though?

Reply:
Ah interesting. Let's not worry about it here, I think we're fine to leave it in if we've used it elsewhere in this file.